.github/workflows/reusable/cached-build/action.yml

```diff
@@ -9,6 +9,7 @@ runs:
         node-version-file: '.nvmrc'
         cache: 'npm'
         cache-dependency-path: package-lock.json
+        registry-url: 'https://registry.npmjs.org'
```

Comment:
According to this document https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry it is mandatory to add this here to properly configure credentials.